tests/page-objects/users/add-user-modal.po.js

```diff
@@ -38,26 +42,84 @@ const getCancelButton = () => {
   return element(by.className('btn cancel'));
 };
 
+const waitForTranslation = (timeout = 50000) => {
+  const description=element.all(by.css('.help-block.ng-scope')).first();
+  const EC = protractor.ExpectedConditions;
+  const usernameHelp = 'This is what you will use to log in to the app.';
+  return browser
+    .wait(
+      EC.textToBePresentInElement(description, usernameHelp),
+      timeout,
+      'Element taking too long to appear in the DOM.Let us retry'
+    )
+    .then(() => {
+      return description.getText().then(text => {
+        if(text === 'user.username.help'){
+          console.log('form not yet translated... ' + text);
+          return browser.sleep(10000);
+        }
+        expect(text).toBe(usernameHelp);
+      });
+    });
+};
+
 module.exports = {
-  submit: () => {
+  submit:  () => {
     helper.waitUntilReady(getSubmitButton());
     getSubmitButton().click();
   },
 
+  waitForFormToDisappear:() => {
```

Comment:
There is a bit of simpler code to handle the element disappearing. 
```
var EC = protractor.ExpectedConditions;
browser.wait(EC.invisibilityOf($('#abc')), 5000);
```

http://www.protractortest.org/#/api?view=ProtractorExpectedConditions.prototype.invisibilityOf